arm-servicebus/2014-09-01/swagger/servicebus.json

```diff
@@ -560,9 +560,343 @@
+          },
+         "AvailabilityStatus ": {
+          "type": "string",
+          "description": "Gets the entity availability status for the queue.",
+          "enum": [
+            "Available",
+            "Limited",
+            "Renaming",
+            "Restoring",
+            "Unknown"
+          ],
+          "x-ms-enum": {
+            "name": "NamespaceType",
+            "modelAsString": false
+          }
+        },
+        "createdAt": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Gets the exact time the message was created."
+        },
+        "DefaultMessageTimeToLive": {
+           "type": "string",
+           "format": "time-span",
+           "description": "Gets or sets the default message time to live value. This is the duration after which the message expires, starting from when the message is sent to Service Bus. This is the default value used when TimeToLive is not set on a message itself."
+          },
+         "DuplicateDetectionHistoryTimeWindow ": {
+           "type": "string",
+           "format": "time-span",
+           "description": "Gets or sets the TimeSpan structure that defines the duration of the duplicate detection history. The default value is 10 minutes.."
+          },
+          "EnableBatchedOperations": {
+          "type": "boolean",
+          "description": "Gets or sets a value that indicates whether server-side batched operations are enabled.."
+        },
+        "EnableDeadLetteringOnMessageExpiration": {
+          "type": "boolean",
+          "description": "Gets or sets a value that indicates whether this queue has dead letter support when a message expires."
+        },
+        "EnableExpress": {
+          "type": "boolean",
+          "description": "Gets or sets a value that indicates whether Express Entities are enabled. An express queue holds a message in memory temporarily before writing it to persistent storage."
+        },
+        "EnablePartitioning": {
+          "type": "boolean",
+          "description": "Gets or sets a value that indicates whether the queue to be partitioned across multiple message brokers is enabled."
+        },
+        "ForwardDeadLetteredMessagesTo": {
+          "type": "string",
+          "description": "Gets or sets the path to the recipient to which the dead lettered message is forwarded."
+        },
+        "ForwardTo": {
+          "type": "string",
+          "description": "Gets or sets the path to the recipient to which the message is forwarded."
+        },
+        "IsAnonymousAccessible": {
+          "type": "boolean",
+          "description": "Gets or sets a value that indicates whether the message is anonymous accessible."
+        },
+        "LockDuration ": {
+           "type": "string",
+           "format": "time-span",
```

Comment:
type string is fine. Take a look at this example https://github.com/Azure/autorest/blob/45ddc1a84c58f5732a2f77418079c9439f39e892/src/dev/TestServer/swagger/body-duration.json#L26

The serialized value will be a string on the wire. However, the format describes the object (Date, Date-Time) from which this needs to be converted and what should be the format of the resulting value.